packages/launch-config/src/debug-config/config.ts

```diff
@@ -73,22 +72,21 @@ export function configureLaunchJsonFile(cwd: string, configOpts: DebugOptions):
         isFioriElement,
         migratorMockIntent
     } = configOpts;
-
-    const projectName = basename(projectPath);
+    const projectName = basename(rootFolder);
     const flpAppIdWithHash = flpAppId && !flpAppId.startsWith('#') ? `#${flpAppId}` : flpAppId;
     const startHtmlFile = flpSandboxAvailable ? testFlpSandboxHtml : indexHtml;
     const runConfig = isAppStudio
         ? JSON.stringify({
               handlerId: FIORI_TOOLS_LAUNCH_CONFIG_HANDLER_ID,
-              runnableId: projectPath
+              runnableId: rootFolder
           })
         : undefined;
     const envUrlParam = getEnvUrlParams(sapClientParam);
 
     const launchFile: LaunchJSON = { version: '0.2.0', configurations: [] };
 
     // Add live configuration if the datasource is not from a metadata file
-    if (datasourceType !== DatasourceType.metadataFile) {
+    if (datasourceType !== ProjectDataSourceType.metadataFile) {
         const startCommand = `${startHtmlFile}${flpAppIdWithHash}`;
         const liveConfig = createLaunchConfig(
```

Comment:
There is already function named `createLaunchConfig` in create.ts. Could you please rename it or just use something like:
```
        const env: LaunchConfigEnv = { DEBUG: '--inspect', FIORI_TOOLS_URL_PARAMS: envUrlParam };
        if (runConfig) {
            env['run.config'] = runConfig;
        }
        const liveConfig = getLaunchConfig(
            `Start ${projectName}`,
            cwd,
            ['fiori', 'run'],
            ['--open', startCommand],
            env
        );
```